Source/DafnyLanguageServer/Workspace/Notifications/VerificationDiagnosticsParams.cs

```diff
@@ -322,7 +322,7 @@ Position Position
       // Ensure that if this is an ImplementationVerificationTree, and children "painted" verified,
       // and this node is still pending
       // at least the first line should show pending.
-      if (range.Start.Line >= 0 && range.End.Line - range.Start.Line + 1 < perLineDiagnostics.Length) {
+      if (range.Start.Line >= 0 && range.End.Line < perLineDiagnostics.Length) {
```

Comment:
I don't understand this change. Is it required to make the test pass?

Reply:
Yes that's the first fix I did before discovering the migration issue. With the migration issue, we shouldn't have this error anymore, but this is still a bug to fix anyway, I just don't know how to reproduce it at this moment.
If you look at it closely, it was requiring the range's length to be less than the array, and then would use the index range.End.line directly. It was throwing an LSP exception. So I wrote it wrong the first time. Fixing this here.